### PR TITLE
#248 Increase polling delay interval

### PR DESCRIPTION
**Pull request by oubchid**

### CODE REVIEW

File: src/esn.inbox.libs/app/app.constants.js [hunk 1]
```diff
@@ -96,4 +96,6 @@ angular.module('esn.inbox.libs')
     isDisplayedByDefault: true
   })
   .constant('INBOX_SIGNATURE_SEPARATOR', '-- \n') // https://tools.ietf.org/html/rfc3676#section-4.3
-  .constant('INBOX_ATTACHMENT_TYPE_JMAP', 'jmap');
+  .constant('INBOX_ATTACHMENT_TYPE_JMAP', 'jmap')
+  .constant('INFINITE_LIST_POLLING_INTERVAL', 10 * 60 * 1000);
```
Comment (huy-ta) [inline src/esn.inbox.libs/app/app.constants.js:100]: I guess this constant is already being defined somewhere else and you're overriding it here, right? IMO, this is not a good way to do it. Why don't you find the original place where it is defined?

Comment (MichaelBailly) [reply src/esn.inbox.libs/app/app.constants.js:100]: @oubchid I'll update the issue description. What we need is 

- keep the polling (INFINITE_LIST_POLLING_INTERVAL) the same for emails list
- increase the polling for mailboxes list

Comment (oubchid) [reply src/esn.inbox.libs/app/app.constants.js:100]: @huy-ta 
It's defined here 
https://ci.linagora.com/linagora/lgs/openpaas/esn/blob/master/templates/js/constants.ejs#L17

I change it here rather to not impact other projects

Comment (chamerling) [reply src/esn.inbox.libs/app/app.constants.js:100]: This one is generated from backend configuration, changing it here means just update the backend configuration.
If you need a new constant, create a new one. Or check where the current one is used

Comment (oubchid) [reply src/esn.inbox.libs/app/app.constants.js]: I create new constant from mailboxes polling delay `INFINITE_MAILBOXES_POLLING_INTERVAL`

**Review by rezk2ll — APPROVED** [review]

**Review by MichaelBailly — APPROVED** [review]

**Pull request merged**